packages/liferay-npm-scripts/package.json

```diff
@@ -19,14 +19,22 @@
 		"@babel/plugin-proposal-class-properties": "7.4.4",
 		"@babel/plugin-proposal-object-rest-spread": "7.4.4",
 		"@babel/preset-env": "^7.4.2",
+		"@storybook/addon-a11y": "^5.1.3",
+		"@storybook/addon-actions": "^5.1.3",
+		"@storybook/addon-knobs": "^5.1.3",
+		"@storybook/addon-viewport": "^5.1.3",
+		"@storybook/react": "^5.1.3",
 		"babel-loader": "8.0.6",
 		"cosmiconfig": "^5.2.1",
 		"cross-env": "^5.2.0",
 		"cross-spawn": "^6.0.5",
+		"css-loader": "^2.1.1",
 		"deepmerge": "^3.0.0",
+		"fs-extra": "^8.0.1",
```

Comment:
Are we sure "fs-extra" is justifying its weight here? It's not obvious to me why we need to create a temporary directory for the stories in the first place — can you help me understand how that need arises?

Reply:
When starting up storybook you need to specify **a directory** which contains the storybook config files like `config.js` and `addons.js`. It currently doesn't seem like there's a way to point to a specific file or configure what the names of specific files are.

fs-extra was added to manipulate directories, but I suppose another possibility is adding custom functions like in the ones in `utils`. I found this gist that does directory manipulation: https://gist.github.com/tkihira/3014700

Reply:
So why can't we just pass `path.join(__dirname, '../storybook')`? It's still not clear to me why we would need to make a copy.

Reply:
(And just to be explicit, you'd obviously need to change the `require('../stories');` thing in the config.js file to load from the current working directory instead of a path relative to the config.js.)

Reply:
Ooo I see, you're right! I was wondering if there was a way to run the command pointing to the storybook directory in `liferay-npm-scripts`. Didn't realize I had it there already.

Since storybook is the one calling `config.js`, would you have any ideas about how I'd load from the current working directory? Possibly somehow import the `cwd` path into it? Or modify the file in the `storybook` script?

Reply:
Oh so changing it to `require(cwd + '/stories');` does work.

Reply:
That shouldn't be happening: other scripts in liferay-npm-scripts rely on `process.cwd()` returning the right directory (ie. the one containing `modules/apps/*/package/json`). Minor nit: we should use `path.join()` instead of `+`.

Reply:
The storybook `config.js` seems to be called within the browser (assuming since `process.title = 'browser'`), so I've set a new environment variable `process.env.STORYBOOK_CWD` within storybook's webpack to use within `config.js`.

(Also updated to use `path.join()`)